cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminRunTime.feature

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Sysadmin
+Feature: Sysadmin - Validate system admin page Run time scenarios
+
+  @Sysadmin
+  Scenario:To verify user should be able to create Namespace successfully in System Admin
+    Given Open Datafusion Project to configure pipeline
+    Then Open "System Admin" menu
```

Comment:
Try using When statements in scenario outlines for all features .

Reply:
Changed to When Statement.